docs/api/fab.md

```diff
@@ -24,603 +17,57 @@ import EncapsulationPill from '@components/page/api/EncapsulationPill';
-    <ion-fab vertical="top" horizontal="end" edge slot="fixed">
-      <ion-fab-button>
-        <ion-icon :icon="person"></ion-icon>
-      </ion-fab-button>
-    </ion-fab>
-
-    <!-- fab placed to the bottom and start and on the bottom edge of the content overlapping footer with a list to the right -->
-    <ion-fab vertical="bottom" horizontal="start" edge slot="fixed">
-      <ion-fab-button>
-        <ion-icon :icon="settings"></ion-icon>
-      </ion-fab-button>
-      <ion-fab-list side="end">
-        <ion-fab-button><ion-icon :icon="logoVimeo"></ion-icon></ion-fab-button>
-      </ion-fab-list>
-    </ion-fab>
-
-    <!-- fab placed in the center of the content with a list on each side -->
-    <ion-fab vertical="center" horizontal="center" slot="fixed">
-      <ion-fab-button>
-        <ion-icon :icon="share"></ion-icon>
-      </ion-fab-button>
-      <ion-fab-list side="top">
-        <ion-fab-button><ion-icon :icon="logoVimeo"></ion-icon></ion-fab-button>
-      </ion-fab-list>
-      <ion-fab-list side="bottom">
-        <ion-fab-button><ion-icon :icon="logoFacebook"></ion-icon></ion-fab-button>
-      </ion-fab-list>
-      <ion-fab-list side="start">
-        <ion-fab-button><ion-icon :icon="logoInstagram"></ion-icon></ion-fab-button>
-      </ion-fab-list>
-      <ion-fab-list side="end">
-        <ion-fab-button><ion-icon :icon="logoTwitter"></ion-icon></ion-fab-button>
-      </ion-fab-list>
-    </ion-fab>
-  </ion-content>
-
-  <ion-footer>
-    <ion-toolbar>
-      <ion-title>Footer</ion-title>
-    </ion-toolbar>
-  </ion-footer>
-</template>
-
-<script>
-import { 
-  IonContent, 
-  IonFab, 
-  IonFabButton, 
-  IonFabList, 
-  IonFooter, 
-  IonHeader, 
-  IonIcon, 
-  IonTitle, 
-  IonToolbar 
-} from '@ionic/vue';
-import { 
-  add, 
-  arrowBackCircle,
-  arrowForwardCircle, 
-  arrowUpCircle,
-  logoFacebook, 
-  logoInstagram, 
-  logoTwitter, 
-  logoVimeo, 
-  person, 
-  settings, 
-  share
-} from 'ionicons/icons';
-import { defineComponent } from 'vue';
-
-export default defineComponent({
-  components: { 
-    IonContent, 
-    IonFab, 
-    IonFabButton, 
-    IonFabList, 
-    IonFooter, 
-    IonHeader, 
-    IonIcon, 
-    IonTitle, 
-    IonToolbar
-  },
-  setup() {
-    return {
-      add, 
-      arrowBackCircle,
-      arrowForwardCircle, 
-      arrowUpCircle,
-      logoFacebook, 
-      logoInstagram, 
-      logoTwitter, 
-      logoVimeo, 
-      person, 
-      settings, 
-      share
-    }
-  }
-});
-</script>
-```
-
-
-</TabItem>
-
-</Tabs>
+Fabs are container elements that contain one or more [fab buttons](/docs/api/fab-button). They should be placed in a fixed position that does not scroll with the content. Fabs should have one main fab-button. Fabs can also contain one or more [`fab lists`](/docs/api/fab-list) which contain related buttons that show when the main fab button is clicked.
+
+## Basic Usage
+
+import BasicUsage from '@site/static/usage/fab/basic/index.md';
```

Comment:
Should the basic example include a list or should we just show a default button? I ask because sometimes they are used to handle a single action: https://m2.material.io/components/buttons-floating-action-button#behavior

Reply:
Good point 🤔 I removed the list from the basic example, and moved the List Side section up to be right under Basic Usage to better introduce the list concept. Let me know if that looks good.